Modix.Services/Mentions/MentionService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Discord;
+using Microsoft.Extensions.DependencyInjection;
+using Modix.Data.Models.Core;
+using Modix.Services.Core;
+
+namespace Modix.Services.Mentions
+{
+    public interface IMentionService
+    {
+        Task<bool> MentionRole(IRole role, IMessageChannel channel, string message = null);
+    }
+
+    [ServiceBinding(ServiceLifetime.Scoped)]
+    internal class MentionService : IMentionService
+    {
+        private readonly IAuthorizationService _authorizationService;
+        private readonly IDesignatedRoleService _designatedRoleService;
+
+        public MentionService(IAuthorizationService authorizationService,
+            IDesignatedRoleService designatedRoleService)
+        {
+            _authorizationService = authorizationService;
+            _designatedRoleService = designatedRoleService;
+        }
+
+        public async Task<bool> MentionRole(IRole role, IMessageChannel channel, string message = null)
+        {
+            if (role is null)
+                throw new ArgumentNullException(nameof(role));
+
+            if (channel is null)
+                throw new ArgumentNullException(nameof(channel));
+
+            if (message is null)
+                message = string.Empty;
+
+            if (role.IsMentionable)
+            {
+                await channel.SendMessageAsync($"You can do that yourself - but fine: {role.Mention} {message}");
```

Comment:
Was this based on an older version of the code? I thought we remove the "You can do that yourself - but fine" wording, because people felt that it was "too sassy"

Reply:
One revert too many